src/Consolonia.AvaloniaEdit/Caret.cs

```diff
@@ -1,3 +1,4 @@
+using System;
 using Avalonia;
```

Comment:
Once application opened - I click "Save". it does not ask for a name (I assume in this situation "Save" must work like "Save As...".